lib/experimental/interactivity-api.php

```diff
@@ -14,7 +14,7 @@
 function gutenberg_interactivity_override_script_module_urls( $url ) {
```

Comment:
Why is this file still in experimental? Is it something that will remain in Gutenberg forever? Should it move to compat folder.

Reply:
This PR will remove that file and move to the external folder:
https://github.com/WordPress/gutenberg/pull/58830